src/main/webapp/js/userMapData.json

```diff
@@ -1078,7 +1245,8 @@
     "Xavier University of Louisiana",
     "Yale University",
     "York College",
-    "Young harris College",
+    "Young Harris College",
+    "Your Group Getaways",
```

Comment:
@fairbet the issue title mentions "Jul-Sep 2016", which excludes June. You should not include whatever was included in the June's list in this PR. In general, when working on an issue, you should not include any unrelated fix unless they are tightly coupled.